test/compile/scale.test.ts

```diff
@@ -389,11 +389,51 @@ describe('Scale', function() {
   });
 
   describe('rangeMixins()', function() {
-    // FIXME
+    describe('row', function() {
+      // TODO:
+    });
+
+    describe('column', function() {
+      // TODO:
+    });
+
+    describe('x', function() {
+      // TODO: X
+    });
+
+    describe('y', function() {
```

Comment:
I don't think this is testing the right thing. We should test what happens if there is only x and x and y.